release.sh

```diff
@@ -232,8 +232,7 @@ release_build_repositories () {
     fi
     log_term 1 "\n${COMMAND} \"${#BUILD_REPOSITORIES[*]}\" repositories at \"${_RELEASE}\"." -e
     log_term 3 "File: '${BASH_SOURCE[0]}' Line: '${LINENO}'"
-    parallel --no-notice --output-as-files --results logs --progress --jobs "${#BUILD_REPOSITORIES[@]}" "${AWS_VAULT_EXEC_MAIN[@]}" "${COMMAND}" '{1}' "${_RELEASE}" ::: "${BUILD_REPOSITORIES[@]}"; _RV=$?
-#    parallel --no-notice --output-as-files --results logs --progress --jobs 4 "${AWS_VAULT_EXEC_MAIN[@]}" "${COMMAND}" '{1}' "${_RELEASE}" ::: "${BUILD_REPOSITORIES[@]}"; _RV=$?
```

Comment:
Commented out code generally doesn't belong in source-control

Reply:
That comment was already removed, reload the PR